### DISCUSSION TITLE
Running without a screen

### DISCUSSION THREAD

**Discussion by brainard92** [junzis/pyModeS#110] [Q&A]
I am looking to run modeslive in the background (as a cronjob, no screen) to use the text file dumped out. Having trouble finding what setting to pass or line(s) to comment out to make that happen. What would be the best way to do this? Thanks!

Edit: I tried simply commenting out various screen-related lines in the main modeslive script, which briefly allows a dump file to be created and written to, but this terminates quickly for reasons I have yet to find. Standard execution of modeslive (with a screen and terminal) is normal on my system (no problems).

**Comment by brainard92** [comment]
Figured it out. In addition to commenting out any/all screen, screen_process, and curses.endwin() lines in modeslive, I needed to comment out these two in decode.py (from pyModeS/streamer).
acs = self.get_aircraft() 
ac_pipe_in.send(acs)

**Comment by junzis** [comment]
Cool, I am glad you figured it out! I will also try to add a new option to make modeslive headless.
